Source/ThirdParty/ANGLE/src/compiler/fuzz/translator_fuzzer.cpp

```diff
@@ -186,6 +191,7 @@ extern "C" int LLVMFuzzerTestOneInput(const uint8_t *data, size_t size)
 
         if (translator == nullptr)
         {
+            sh::Finalize();  // WebKit change.
```

Comment:
Maybe just remove the comments..
We manage the changes other way, so maybe just add the code as you would in a normal and I can upstream them.

Reply:
Will do.  Thanks!  (I was trying to follow the `libwebrtc` rules here.)